backend/handlers/handlers.go

```diff
@@ -812,11 +812,29 @@ func (handlers *Handlers) getAccountSummary(_ *http.Request) (interface{}, error
 // getSupportedCoinsHandler returns an array of coin codes for which you can add an account.
 // Exactly one keystore must be connected, otherwise an empty array is returned.
 func (handlers *Handlers) getSupportedCoinsHandler(_ *http.Request) (interface{}, error) {
+	type element struct {
+		CoinCode      coinpkg.Code `json:"coinCode"`
+		Name          string       `json:"name"`
+		CanAddAccount bool         `json:"canAddAccount"`
+	}
 	keystores := handlers.backend.Keystores().Keystores()
 	if len(keystores) != 1 {
 		return []string{}, nil
 	}
-	return handlers.backend.SupportedCoins(keystores[0]), nil
+	keystore := keystores[0]
+	var result []element
+	for _, coinCode := range handlers.backend.SupportedCoins(keystore) {
+		coin, err := handlers.backend.Coin(coinCode)
+		if err != nil {
+			return nil, err
```

Comment:
Just skip it? Seems less impactful: I'd still want to be able to add an account for any other supported coin.

Reply:
I like it!